### PR TITLE
math-magiciens-Setup

### PR DESCRIPTION
**Pull request by Patfarmurs**

### CODE REVIEW

**Comment by TedLivist** [comment]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Patfarmurs,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly update your README with relevant information about the project, instead of leaving it as a dummy file. Be sure to remove the logo (since it is broken) or add yours (instead of the Microverse logo). Also remove the muted texts that served as hints.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rloterh — COMMENTED** [review]
Hi Patrick, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that the _Future Features_ section of your ReadMe includes 1-3 potential features to be added later. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Patfarmurs** [comment]
Why are you guys wasting my time??? It's just a setup come on!!!

**Comment by rloterh** [comment]
Hi @Patfarmurs,

I want you to understand that minor things really matter in the industry. It's a good thing that we get adapted to paying attention to details, even minor details at this point, which we'll benefit from eventually once we make it to the industry level.  I believe you'll now agree with me that no one is wasting your time, but rather a means of building good habits for the future!

Cheers! 😃

File: README.md [hunk 1]
```diff
@@ -58,64 +22,44 @@ After you're finished please remove all the comments and instructions!
...
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
```
Comment (amejid) [inline README.md:46]: - [ ] Kindly remove all lines that start with `>` as they are instructions on what to do. Please delete them after you are done editing the document.

**Example**
image

File: README.md [hunk 2]
```diff
@@ -205,21 +125,14 @@ Example:
...
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
```
Comment (amejid) [inline README.md:138]: - [ ] I suggest you remove the placeholder text and put some information here.

**Review by amejid — COMMENTED** [review]
Hi @Patfarmurs, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README. It would be great to implement the suggestions not to waste your time and ours.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Patfarmurs** [comment]
No way, It's just a set-up project, did you even read the project instructions? you guys lack empathy.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Patfarmurs, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Patfarmurs** [comment]
Thanks a lot  @ShadyShawkat

**Pull request merged**